types/ember__test-helpers/index.d.ts

```diff
@@ -189,8 +189,21 @@ declare module '@ember/test-helpers/settled' {
 declare module '@ember/test-helpers/setup-context' {
     import Resolver from '@ember/application/resolver';
 
+    interface Owner {
+      lookup: (name: string) => unknown;
+      register: (name: string, mockService: unknown) => void;
+    }
```

Comment:
`Owner` should already have types, no?

Basically, this thing is "owner":

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/15fe0696be464ec1f7cc97ed42f1ec35bc4476b5/types/ember__engine/instance.d.ts#L9